src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
```

Comment:
Don't use verbs for class/interface names. Consider renaming `IsoscelesTrapezoid` to a noun form if possible, although in this case, it might be acceptable as it is a specific shape name.